src/controllers/notifications.js

```diff
@@ -1,67 +0,0 @@
-'use strict'
```

Comment:
That's assuming we're talking about the client who sent the transfer originally, and that that client has state. If you're listening for incoming transfers you'll have no idea you missed something if you never heard about it in the first place. We also shouldn't assume that clients or connectors are keeping state. Many will rely on the ledger for state.